docs/api/en/extras/ShapeUtils.html

```diff
@@ -39,8 +39,8 @@ <h3>[method:Boolean isClockWise]( pts )</h3>
 
 		<h3>[method:Array triangulateShape]( contour, holes )</h3>
 		<p>
-		contour -- 2D polygon.<br />
-		holes -- array of holes<br /><br />
+		contour -- 2D polygon. An array of THREE.Vector2()<br />
+		holes -- An array of THREE.Path()<br /><br />
```

Comment:
I'm afraid this line is incorrect. `holes` is an array that holds arrays of `Vector2`. Meaning for each hole definition a separate array.

Reply:
Oh, that makes a lot of sense! My bad. I will change that as soon as I can.